doc/02-installation.md

```diff
@@ -125,14 +125,15 @@ wget https://packages.icinga.com/subscription/rhel/ICINGA-release.repo -O /etc/y
 If you are using RHEL you need to additionally enable the `codeready-builder`
 repository before installing the [EPEL rpm package](https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F).
 
-#### RHEL 8
+#### RHEL 8 or Later
 
 ```bash
 ARCH=$( /bin/arch )
+OSVER=$( . /etc/os-release; echo "${VERSION_ID%%.*}") )
```

Comment:
That's too many `)` now.

Reply:
Damnit... fixed that and rebased the branch.